docs/monthly-meeting/2024-02.md

```diff
@@ -0,0 +1,92 @@
+    1. **Should we add more examples in general?**  Are they preferable to prose?
+    2. Quoting from [this comment](https://github.com/python/cpython/pull/111743#issuecomment-1908471823), especially the last part:
+        > [...] **Should we have some convention on where to put examples?**
+        >
+        > We don't want to end up in a situation where a page uses collapsible sections, another has [examples at the top](https://docs.python.org/3/library/json.html), another [at the bottom](https://docs.python.org/3/library/re.html#regular-expression-examples), another [in the middle](https://docs.python.org/3/library/string.html#format-examples), another [in a separate page](https://docs.python.org/3/howto/logging-cookbook.html), etc.  If we agree on some conventions, it will also be easier for users to set their expectations and know where to look.
+
+        See also [the Discord thread](https://discord.com/channels/935215565872693329/1199481017464008734) and a [related issue](https://github.com/python/cpython/issues/106318#issuecomment-1906642662)
+
+    [Daniele] Examples work best as relief from dense abstract docs. They should be boring, and simple. You don't need to be exhaustive; the example is only a handhold.
+    [Ezio] Maybe we should have simple examples inline, and a section with longer examples at the bottom, with links pointing to it.
+    [Daniele] An example section sounds like the beginnings of how-to page.
+    [Ned] I like examples for clarifying the text in the reference sections. Interspersing examples into the reference would help, at least in some places.
+    [CAM] Tutorials vs. how-tos - that depends on the purpose of the examples
+    [Ned] How do we turn this thinking into actionable items people can help with?
+    [Ezio] We should start adding examples.
+
+    [Ezio] We should also figure out what to do about collapsible sections.
+    [Ned] What problem are collapsible sections solving? Do we have too many examples that users will want to ignore? (In my docs I use tabs, but not collapsible sections)
+    [Ezio] If we agree to add basic examples inline, but bigger examples on the bottom, we won't need collapsible sections.
+    [Carol] So: yes examples, yes inline. We can pause discussions about collapsible sections; if the examples get too long we should think about spinning them out into their own howto pages. **Consensus Recap**
+    [Ezio] We could create a custom Sphinx directive that creates a link to the "Examples" section too.  It should be unobtrusive and recognizable (it could have an icon, and a short text like "See more examples.").
+
+    [Ezio] Would it make sense to do different things based on the builder, e.g. collapsible sections for HTML and something else for PDF?
+    [CAM] Yes. Some current directives for collapsible sections do that.
+
+    [Ned] re. pages being too long, we might switch to having a page for each function
+    [Carol] we could run a cron job to count lines
```

Comment:
```suggestion
    [Ezio] We should also figure out what to do about collapsible sections.
    [Ned] What problem are collapsible sections solving? Do we have too many examples that users will want to ignore? (In my docs I use tabs, but not collapsible sections)
    [Ezio] They allow us to to add more examples inline without making the page too long and more difficult to navigate, but they don't work with non-HTML builders.

    [Ned] re. pages being too long, we might switch to having a page for each function
    [Carol] we could run a cron job to count lines
    [Ezio] if we count code lines we can determine pages with not enough and too many examples

    [Ezio] Would it make sense to do different things based on the builder, e.g. collapsible sections for HTML and something else for PDF?
    [CAM] Yes. Some current directives for collapsible sections do that.
    
    [Ezio] If we agree to add basic examples inline, but bigger examples on the bottom, we won't need collapsible sections.
    
    [Carol] So: yes examples, yes inline. We can pause discussions about collapsible sections; if the examples get too long we should think about spinning them out into their own howto pages. **Consensus Recap**
    [Ezio] We could also create a custom Sphinx directive that creates a link to the "Examples" section.  It should be unobtrusive and recognizable (it could have an icon, and a short text like "See more examples.").
```

I tried to reorganize this section so that it makes more sense for people that weren't at the meeting, but if this sort of historical revisionism is not allowed let me know.